presto-hive/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -611,7 +640,8 @@ public void dropColumn(String databaseName, String tableName, String columnName)
                         .withDatabaseName(databaseName)
                         .withTableName(tableName)
                         .withExpression(expression)
-                        .withNextToken(nextToken));
+                        .withNextToken(nextToken)
+                        .withSegment(segment));
```

Comment:
move before `withNextToken`. Does `withSegment` accept `null`?

Reply:
Yes, it accepts null.